src/infrastructure/validation/logger/ConsoleLogger.java

```diff
@@ -110,17 +121,33 @@ synchronized public void log(ModuleID moduleIdentifier, LogLevel level, String m
 		LocalDateTime now = LocalDateTime.now();
 		String formatDateTime = now.format(timeStampFormat);
 
-		String logTimeStamp = "["+formatDateTime+"]";
+		String logTimeStamp = "";
 
-		String logModulePart = "["+moduleIdentifier.toString()+"]";
+		String logModulePart = "";
 		
-		String logLevelPart = "["+level.toString()+"]";
+		String logLevelPart = "";
+		
+		if(!inTestMode) {
+
+			logTimeStamp = "["+formatDateTime+"]";
+			
+			logModulePart = "["+moduleIdentifier.toString()+"]";
+			
+			logLevelPart = "["+level.toString()+"]";
+		}
 		
 		String logMessage = logTimeStamp+logModulePart+logLevelPart+message;
 
 		printToConsole(level, logMessage);
```

Comment:
Have a check to see if explicitly a moduleID was supplied as a parameter. We do not want module logging calls being logged during the test run. If such an identifier was given, just don't log anything

Reply:
Now that is just hardcoding stuff
but ok
Fine, I can try that

Reply:
added a way to skip other modules. let me know in case you know of any better method
maybe useful ref: [break/return in java][link]

[link]:<https://stackoverflow.com/questions/7937029/how-to-break-out-or-exit-a-method-in-java>